packages/server/lib/browsers/firefox.ts

```diff
@@ -579,3 +579,7 @@ export async function open (browser: Browser, url: string, options: BrowserLaunc
 
   return browserInstance
 }
+
+export async function closeExtraTargets () {
```

Comment:
Can we add details on why this isn't supported and/or what we would still need to do or support this? same with webkit? 

you've spent alot of time on this so your knowledge likely be lost on the next developer who has to look at this if we dont add some notes

Reply:
Added comments in https://github.com/cypress-io/cypress/pull/28204/commits/5f8fbe399430e385e588503f49d5f33599a3cebb